Documentation/Eureka 2.0 Migration  Guide.md

```diff
@@ -32,12 +35,10 @@ There are also some breaking changes related with deprecated API:
 
 Removed APIs:
 
-* `PostalAddressRow` and `ImageRow` was deleted.
-*  row has been deleted. You can find it and many other custom rows at EurekaCommunity [organization account](https://github.com/eurekaCommunity).
-
-`highlightCell` and `unhighlightCell` callbacks were deleted, now we should use `row.isHighlighted` from cell update to check from highlighted status and make UI modification according its value.
+* `PostalAddressRow` and `ImageRow` was deleted. You can find them and many other custom rows at EurekaCommunity [organization account](https://github.com/eurekaCommunity).
```

Comment:
They are currently in progress. We will soon add them there. I believe the ImageRow has also been added to the Examples project.